qiskit_ibm_runtime/base_primitive.py

```diff
@@ -20,10 +20,13 @@
 from dataclasses import asdict
 import warnings
 
+from qiskit_aer import AerSimulator
```

Comment:
Aer is not a required package, so this import be wrapped with try/except

Reply:
Added `try` and raise exception when attempting to run with a simulator and not aer.
Also added `skip` for relevant tests.
Please have a look if it is done properly.